src/main/scala/nl/knaw/dans/easy/fedora2vault/DDM.scala

```diff
@@ -302,8 +302,8 @@ object DDM extends DebugEnhancedLogging {
   }
 
   private def isDdmId(bi: BasicIdentifier): Boolean = {
-    // these ID's are generated by easy-ingest-flow/EASY-I, not converted from DDM
-    !Seq("PID", "DMO_ID", "AIP_ID").contains(bi.getScheme)
+    // generated into EMD by easy-ingest-flow/EASY-I, not schema valid in DDM
```

Comment:
do you mean to say that with these identifiers in dataset.xml the file is no longer valid?

Reply:
When omitting either one from this list we would indeed get 
```
org.xml.sax.SAXParseException; lineNumber: 18; columnNumber: 44; cvc-elt.4.2: Cannot resolve 'id-type:PID' to a type definition for element 'dct:identifier'.
org.xml.sax.SAXParseException; lineNumber: 19; columnNumber: 47; cvc-elt.4.2: Cannot resolve 'id-type:DMO_ID' to a type definition for element 'dct:identifier'.
```
at   https://github.com/DANS-KNAW/easy-fedora2vault/blob/d7e91a52a1fb8bad3979bce6ab35a20afce0359c/src/test/scala/nl/knaw/dans/easy/fedora2vault/DdmSpec.scala#L109

Reply:
Ah, `id-type:PID` should have been `id-type:URN`

Reply:
do you know if the fedora-id is in the resulting dataset.xml?

Reply:
Doesn't look like that https://github.com/DANS-KNAW/easy-fedora2vault/blob/016caf55f2dfa07800a1d5749e672b8d4540333a/src/main/scala/nl/knaw/dans/easy/fedora2vault/DDM.scala#L304-L307